Oracle.NoSQL.SDK/src/Oracle.NoSQL.SDK.csproj

```diff
@@ -9,6 +9,7 @@
     <Copyright>Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.</Copyright>
```

Comment:
This change is to add package read me file, done some time ago. I forgot about it and it got pulled into this PR.  This should be fine to check in though, as it should help display contents of the README on the SDK nuget page.